source/write-scripts/considerations.txt

```diff
@@ -0,0 +1,27 @@
+.. _mongosh-script-limitations:
+.. _mongosh-script-considerations:
+
+=====================
+Script Considerations
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/fact-call-limitations.rst
+
+Code Objects
+~~~~~~~~~~~~
+
+.. include:: /includes/scripting/fact-code-objects.rst 
```

Comment:
I’m not sure I really get why these things are being listed here. The `Code` object note in particular is very specific to usage of the `Code()` class and not related to scripting. (I don’t think we even document `Code()` explicitly?)